examples/images_contours_and_fields/contourf_demo.py

```diff
@@ -6,6 +6,7 @@
 How to use the `.axes.Axes.contourf` method to create filled contour plots.
 """
 import numpy as np
+import matplotlib as mpl
```

Comment:
The disadvantage is the extra mpl import that isn't used anywhere else.  Is it possible to alias to `plt.colormaps`?

Reply:
I'm open to that discussion.

**Pro:** `plt.colormaps` would save the `mpl` import.

**Con:** While we want to deprecate `cm.get_cmap`, I'm not so sure about `plt.get_cmap`. Pyplot is very function centered. And having both the functions and a `colormaps` registry in the long run, is a redundancy we need to question if it's worth it.

After all, pyplot is mainly for "simpler" plots, to which advanced colormap functionality does not necessarily count.

In the more complex examples, we have `plt` only around because of `plt.subplots` and `plt.show()` which mid-term may be replaced by another API, so that we don't import pyplot there anymore at all. `colormaps` should definitively be part of that new interface. I'm unsure if we then still would want it in pyplot.

That probably needs a dev call discussion. For now, I've gone the way of least API change. We can easily add it later, but if I introduce it, removing again would be pain.